pkg/console/operator/operator.go

```diff
@@ -110,9 +116,10 @@ func NewConsoleOperator(
 		serviceClient:    corev1Client,
 		deploymentClient: deploymentClient,
 		// openshift
-		routeClient:   routev1Client,
-		oauthClient:   oauthv1Client,
-		versionGetter: versionGetter,
+		routeClient:         routev1Client,
+		oauthClient:         oauthv1Client,
+		consolePluginClient: consolePluginClient,
```

Comment:
you only appear to be doing a `List()` over console plugins, you should use a lister (it caches the data so that you don't need to do live requests to the API server)